state/aws/dynamodb/dynamodb_test.go

```diff
@@ -232,6 +313,45 @@ func TestSet(t *testing.T) {
 		err := ss.Set(req)
 		assert.NotNil(t, err)
 	})
+	t.Run("Un-successfully set item with ttl (invalid value)", func(t *testing.T) {
```

Comment:
Can you please update every occurrence of `un-successfully` to be `unsuccessfully`?